### ISSUE TITLE
Whiteship navigation console can be used to easily locate the Nuke-Ops shuttle.

### ISSUE THREAD

**Issue by GravitronBlue** [tgstation/tgstation#32370]
EDIT: MSO has asked that I cut out the old format template text and provide a better explanation along with some screenshots.

~~title,~~
~~What happened instead: Crew finds and uses the navigation console on the white ship to hunt down and raid the syndicate infiltrator at little risk with no repercussions.~~

~~Why is this bad/What are the consequences: Nuclear Operatives gamemode becomes less about guarding a disk and gets replaced by cheesy tactic users stealing the nuke as a result of "emergent gameplay."~~

~~Steps to reproduce the problem: Spend 5 minutes searching for the white ship in space, use the navigation console on it to locate the syndicate shuttle in 10 seconds. Proceed to raid the shuttle and fly into deep space with their nuke/major victory aboard.~~

Navigation consoles can see the nuke-ops ship/syndicate-infiltratior landing zone, letting the crew confirm nuke-ops before they even reach the station.

My thoughts are that you shouldn't be able to see the landing zones for other ships, and perhaps not even see the syndicate ship itself with a navigation console.

What ends up happening on occasion is some crew on the abandoned ship sit around waiting for war-ops to board the station so they can go steal the nuke while the Ops are still hunting the disk.

I see this is a significant balance issue, as by just having the abandoned ship you're able to patrol/pan around the station for free and detect/meta the Nuke-Ops by finding their ship or landing zone. Doing so spoils the round because people can end up "accidentally" doing this by looking around the station, which is a whole lot easier to justify than going out in space intentionally looking for the syndicate shuttle.

Performing and exploiting this is all about being ready before the Ops are, which means hunting down the abandoned ship with a GPS, EVA gear, and some luck. Once you're set just pan around until you see something like this:
https://i.imgur.com/eUlzGeh.png
If you have well pat yourself on the back and ~~calmly PDA the captain/HoS/AI that you've spotted nuclear operatives~~ shout in all caps with as many exclamation points as possible about ops. After that just watch the Ops rush in trying to bag the disk before the whole crew arms up. With all the Ops busy stationside you're basically home free raiding their ship now, just park next to it like so:
https://i.imgur.com/1bRIBo9.png
Since the syndicate are so clumsy they never added any sort of access restriction to the windoor so you can just open it and run off with the nuke behind it.
https://i.imgur.com/NDKSGD8.png
Enjoy your redtext operatives!

Bagil
Revision: 741016
Round ID: 28271

Full explanation below

Pretext:
Apologies if this ends up reading like a needlessly long, salty, "I DED," sort of report.

Story:
This report comes following a round of War-Ops where the crew rushed the white ship and then located the Nuke-Ops shuttle and stole the nuke from it, all while the ops were busy inside the station. Lone Op was eventually successful in acquiring the nuke disk, but later lost it during the raid to reclaim the nuke because the white ship was moved during the fight, warping the disk back to the station, ruining the victory that Ops should've had right then.

Issue:
A problem with this is that it's a clear rule 2 precedent 5 violation, that "Station crew are currently not allowed to board or search for the syndicate shuttle during nuke op rounds, unless they specifically follow an operative or a pinpointer onto the shuttle. It is better to adminhelp first in this situation, so admins can confirm that you found it legitimately."
https://tgstation13.org/wiki/Rules#Rule_2_Precedents.

I mentioned this rule in deadchat but was told that the rule was outdated.
ADMIN(Thunder12345) says, "That's pretty outdated by now because of the navigation consoles"
If this is true then I highly suggest removing or changing the precedent.

Regardless of the rule being outdated or not with the introduction of navigation consoles; nuke stealing with the intent to make the round unwinnable by the Ops is a problem. The Operative shuttle used to be both indestructible and a secure place protected by the rules to prevent nuke snatching. Neither of which seems to be true at this time, thanks to it having a breakable/weldable hull (Despite having an indestructible Opshuttle wall typepath! Though I hear WJohn is planning a kind of reinforced plastitanium wall.) and turrets that can be attacked from safe angles.

Reasoning:
The main reason for this is that it's the crew's job is to secure/protect the disk, kill the Ops, and evacuate.
If we're going to allow people to instead secure their own ship, kill the syndicate shuttle, evacuate into deep space with the syndicate nuke and overall take the fight to the syndicate, it should at least be harder than playing defensively, not as a cheaper and more effective way to win by raiding an empty shuttle.

Credits:
If you've read this far then I thank you for reading all this and the work making TG a fun and enjoyable game of spessmans.

TLDR:
Infiltrator is easy to find, easy to steal the nuke from and the rules preventing such no longer apply.

**Comment by Bawhoppen** [comment]
But that's already the case with a jetpack anyways? It takes about 1 minute to find it assuming you're the most unlucky guy on the station. We have a rule for it to deal with people from doing that, so I imagine the only sensible solution is to apply it to this too.

**Comment by ninjanomnom** [comment]
This is a problem with the entire concept of custom placement of shuttles using advanced cameras. It was a regrettable move to allow it as a precedent.

**Comment by WJohn** [comment]
You can already do this with a jetpack, and even more easily if you have a beam rifle because that gives you even more vision. The problem is less about being able to find the syndie infiltrator, and more about being able to steal the nuke.

I suggest we make it so that:
1. The nuke automatically teleports back to its place in the syndie ship if moved off the z level or destroyed.
2. The syndie pinpointer is changed so using it changes it mode to one of four options (off, nuke disk, syndie nuke, syndie ship).
3. Nuke op's code is identical to the station's nuke, so it is possible as a backup to detonate the station's nuke if yours is compromised.
4. Projectile mechanics are changed so they can actually hit people nearby if they're fired from inside a wall like with turrets, because right now you can hang around right next to a turret (a tile beneath it) and you're completely, totally safe. If that's not an option, maybe make them fire special high damage x-ray lasers.

**Comment by evsey9** [comment]
Just apply a weekban on everyone who does this for bug exploit and breaking another rule

**Comment by WJohn** [comment]
Admins enforcing shit is not a good way to prevent something from being ruined. We had to add a mechanic where the nuke disk teleports back onto the station if moved to another z level, as banning every captain who does that right off the bat would be stupid and annoying.

**Comment by Spaceinaba** [comment]
lets remove jetpacks and flight suits then
also remove throwing objects and shooting to change direction in space
that way nobody will be able to get to the op ship

**Comment by TheWiznard** [comment]
what if you just disable non syndi nav ship consoles when it's nuke? something about syndicate malware?

**Comment by Bawhoppen** [comment]
That would be very meta-able
The only solution is a rules implementation or removal of the nav console in its current form

**Comment by jughu** [comment]
To be fair the turrets dont cover the left and right wing only the darn corners

**Comment by jughu** [comment]
Making it easy to steal the nuke without any effort

**Comment by WJohn** [comment]
It's actually better that they cover corners, because then at least they can hit SOMETHING sometimes. Them being in the middle of walls would make them unable to shoot anything at all unless it was like very nearly a straight cardinal line shot from them.

**Comment by MimicFaux** [comment]
What if the turrets were on little gun platforms that stuck out from the ship itself?

**Comment by MrStonedOne** [comment]
There needs a hell of a lot more explimation on this issue report

How did they use the white ship shuttle to locate the nuke ops shuttle.

Give me screens showing the process

**Comment by MrStonedOne** [comment]
>What did you expect to happen:

>What happened instead:

>Why is this bad/What are the consequences: 

>Steps to reproduce the problem:

Remove this bullshit from the issue report. Talk in complete sentences.

**Comment by WJohn** [comment]
Basically the white ship, and really all ships with navigation consoles, provide extended view range with turf view only (like mesons). This makes it really easy to move it around the station's outside border in a matter of moments and notice things like syndicate shuttles. You can also do this equally easy with, say, particle beam guns and a jetpack because those give you something like 40x40 view range or more.

Couple this with the fact that nuke ops often WANT their ship to be near the station to prevent idiot new ops from flying off to deep space because they don't know what a pinpointer is, and you've got yourself a bad case of effortlessly discovered syndie ship that can be even more effortlessly broken into with the shitty turret sightlines.

The best fix for this honestly is to make stealing the nuke not the end of the world for the ops, I've already stated above some ways to help with that.

**Comment by GravitronBlue** [comment]
Rewrote the issue report and provided some screenshots as requested by MSO.

**Comment by MrStonedOne** [comment]
Thank you
